src/main/java/Calculator.java

```diff
@@ -48,7 +47,12 @@ private double divide(double x, double y) {
         return x / y;
     }
 
+    public double clearCalculator() {
```

Comment:
Can we modify the clear function to return void? The function always returns 0 and the return value is not used in the main function of the main class.

Reply:
And possibly clear the console ?

Reply:
Great job separating out the clearCalculator() function for readability!